### PR TITLE
add: eviction policy Dragonfly

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

File: docs/products/dragonfly/howto/eviction-policy-df.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Data eviction policy in Aiven for Dragonfly
+sidebar_label: Data eviction policy
+---
+
+Aiven for Dragonfly® optimizes cache memory management with a data eviction policy, ensuring efficient memory usage and optimal performance.
```
Comment (bogdanp05) [inline docs/products/dragonfly/howto/eviction-policy-df.md:6]:
```suggestion
Aiven for Dragonfly® optimizes cache memory management with a low-overhead data eviction policy.
```
sounds better imo. Original sounds like we're trying too hard to oversell it. But up to you.

File: docs/products/dragonfly/howto/eviction-policy-df.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  significance.
+
+This dual-buffer approach ensures resilience against fluctuating access patterns,
+maintaining cache efficiency without additional memory overhead per item.
+It's a key factor in Dragonfly's ability to keep valuable data accessible while managing
+less critical data efficiently.
+
+## Enable `cache_mode` via Aiven Console
+
+By default, `cache_mode` in Aiven for Dragonfly is disabled, providing basic caching.
```
Comment (bogdanp05) [inline docs/products/dragonfly/howto/eviction-policy-df.md:27]: `cache_mode` is indeed disabled by default, but that means that no caching is used. So as soon as `maxmemory` is reached, writes will fail with an OOM exception.

**Review by bogdanp05 — CHANGES_REQUESTED** [review]

File: docs/products/dragonfly/howto/eviction-policy-df.md (hunk 1 above)
Comment (harshini-rangaswamy) [reply docs/products/dragonfly/howto/eviction-policy-df.md]: Updated to the suggestion.

File: docs/products/dragonfly/howto/eviction-policy-df.md (hunk 2 above)
Comment (harshini-rangaswamy) [reply docs/products/dragonfly/howto/eviction-policy-df.md]: Updated text.

**Review by bogdanp05 — APPROVED** [review]
lgtm

**Review by ArthurFlag — APPROVED** [review]

**Pull request merged**